homeassistant/components/acmeda/base.py

```diff
@@ -0,0 +1,60 @@
+"""Base class for Acmeda Roller Blinds."""
+import aiopulse
+
+from homeassistant.core import callback
+from homeassistant.helpers import entity
+
+from .const import DOMAIN, LOGGER
+
+
+class AcmedaBase(entity.Entity):
+    """Base representation of an Acmeda roller."""
+
+    def __init__(self, hass, roller: aiopulse.Roller):
+        """Initialize the roller."""
+        self.hass = hass
+        self.roller = roller
+
+    async def async_added_to_hass(self):
+        """Entity has been added to hass."""
+        self.roller.callback_subscribe(self.notify_update)
+
+    async def async_reset(self):
```

Comment:
Rename this to `async_will_remove_from_hass`.

Reply:
Changed.